filebeat/docs/filebeat-options.asciidoc

```diff
@@ -156,4 +156,4 @@ include::inputs/input-udp.asciidoc[]
 
 include::inputs/input-unix.asciidoc[]
 
-include::../../x-pack/filebeat/docs/inputs/input-websocket.asciidoc[]
+include::../../x-pack/filebeat/docs/inputs/input-streaming.asciidoc[]
```

Comment:
I think this line needs moved up to ~L150.

Reply:
Yes

Reply:
I miss mechanically generated documentation rendering.